packages/shell-api/src/collection.ts

```diff
@@ -2056,4 +2056,66 @@ export default class Collection extends ShellApiWithMongoClass {
       ...options
     });
   }
+
+  @serverVersions(['6.0.0', ServerVersions.latest])
+  @returnsPromise
+  @apiVersions([])
+  async getSearchIndexes(options?: any): Promise<Document[]> {
```

Comment:
```suggestion
  async getSearchIndexes(options?: Document | string): Promise<Document[]> {
```

(I guess? And I guess we’re not doing `(name, options?)` then?)